docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -98,16 +98,30 @@ When setting the `uv_dose_type` configuration option to `UVDoseType.calculated`,
    "Relative lamp output", ":math:`\frac{S}{S_0}`", "relative_lamp_output", None, None
    "Number of banks", ":math:`N_{bank}`", "num_of_banks", None, None
 
+When setting the ``has_aop`` configuration option to ``True``, there are 2 additional variables that must be fixed.
```

Comment:
For multiple species, I think (j+1) additional variables need to be provided if describe precisely. `photolysis_rate_constant` or `reaction_rate_constant` has j, and only need 1 `hydrogen_peroxide_conc` or `second_order_rate_constant` (other can be derived). However, for here, I think it is more easy for understanding to use 2 instead of (j+1), and the variable simply represents the notation listed below.